migrations/versions/aa7f37526ffc_.py

```diff
@@ -0,0 +1,63 @@
+"""empty message
```

Comment:
squash multiple migrations into one within a PR whenever possible. 
The less migrations we have the cleaner the history